src/Documentation/resources/Best_Practices.md

```diff
@@ -0,0 +1,422 @@
+# Best Practices
+
+Orleans was designed in 2014 with the intention of removing low-level
+abstractions from actor model frameworks. Grains (actors stored in memory) are
+used to maintain the lifecycle among other features. Orleans is based on the
+actor model and fits some projects better that others.
+
+## Orleans should be considered when:
+
+-   Significant number (hundreds to millions) of loosely coupled entities
+
+    -   Example: Internet of Things (IoT) – individual devices turning on/off
+
+-   Entities are small enough to be single-threaded
+
+    -   Example: Determine if stock should be purchased based on current price
+
+-   Workload is interactive
+
+    -   Example: request-response, start/monitor/complete
+
+-   More than one server is expected or may be required\\
+
+    -   Orleans runs on a cluster which is expanded by adding servers to expand
+        the cluster
+
+-   Global coordination is not needed or on a smaller scale between a few
+    entities at a time
+
+    -   Orleans maintains its clusters and talks to other clusters. Each cluster
```

Comment:
I don't see how this illustrates the topic. Maybe something along the lines of:
"Scalability and performance of execution is achieved by parallelizing and distributed a large number of mostly independent tasks with no single point of synchronization." instead?

Reply:
corrected